### ISSUE TITLE
Translation of files to have a multilingual platform

### ISSUE THREAD

**Issue by xeviscc** [CloudOpting/cloudopting-manager#241]
We have to translate all the sentences from English to all other languages in order to have a multilingual platform

**Comment by gioppoluca** [comment]
Can you make a list of the files we need to translate?
So we can assign them to people?

**Comment by xeviscc** [comment]
All files in https://github.com/CloudOpting/cloudopting-manager/tree/master/rest-component/src/main/webapp/i18n/en

**Comment by gioppoluca** [comment]
@ClaudioSola you take care of the italian translation

**Comment by xeviscc** [comment]
Other languages remain to be translated.

**Comment by guidospadotto-profesia** [comment]
@xeviscc assign this issue to a native speaker of the "other" languages. @cpopsa ?

**Comment by xeviscc** [comment]
@cpopsa, could you translate to RO? And when finished, assign it to me again?

**Comment by xeviscc** [comment]
Please take the json files from en folder. The files in other languages can be old and some strings may not be there.

**Comment by xeviscc** [comment]
Noticed that some string are not in language json. Depends on #268

**Comment by xeviscc** [comment]
All texts are ready to be translated.

**Comment by xeviscc** [comment]
Check versions in English if something is not correctly shown.

**Comment by ciprianpavel** [comment]
@xeviscc You **MUST **create all the text in EN language **THEN ** create issues to have the texts translated.
If during translation process you change the files will mess up the process.

**Comment by xeviscc** [comment]
Multilingual platform is working. Each language has its own issue in order to keep track of it.
